README.md

```diff
@@ -47,6 +48,35 @@ config = Config(
 vrt = VisualRegressionTracker(config)
 ```
 
+#### Or, as JSON config file `vrt.json`
+```json
+{
+    "apiUrl"="http://localhost:4200",
+    "project"="Default project",
+    "apiKey"="tXZVHX0EA4YQM1MGDD",
+    "ciBuildId"="commit_sha",
+    "branchName"="develop",
+    "enableSoftAssert"=false
+}
+```
+```python
+vrt = VisualRegressionTracker()
+```
+
+#### Or, as environment variables
+```sh
+VRT_APIURL="http://localhost:4200" \
+VRT_PROJECT="Default project" \
+VRT_APIKEY="tXZVHX0EA4YQM1MGDD" \
+VRT_CIBUILDID="commit_sha" \
+VRT_BRANCHNAME="develop" \
+VRT_ENABLESOFTASSERT=1 \
```

Comment:
cannot get this env propperty to override config file with `false` value
I'm getting in console like string and I guess it's treated as `true`
```
Config(apiUrl='http://localhost:4200', ciBuildId='commit_sha', branchName='develop', project='Default project', apiKey='0TK0P0NQP6MNFQQPTYYBN27JRAA5', enableSoftAssert='False') None
```

Reply:
🤦  fixed